### ISSUE TITLE
Enhancement request: How to get an object by primary key when it is a composite key

### ISSUE THREAD

**Issue by paxet** [ponyorm/pony-doc#7]
When using [Composite primary keys](https://docs.ponyorm.com/api_reference.html#composite-primary-and-secondary-keys) for an entity and then `entity.get_pk()` it returns a tuple.
How to [retrieve object](https://docs.ponyorm.com/firststeps.html#getting-objects) from database with that tuple?

I'm seeking something like this:
```
# Define entity
MyEntity(db.Entity):
  a = Required(str)
  b = Required(str)
  PrimaryKey(a, b)

# Create and Save object to database
ent = MyEntity()
ent.a = 'Key1'
ent.b = 'Key2'
pony.orm.commit()

# Save reference
classname = ent.__class__.__name__
primary_key = ent.get_pk()

# Get from database by primary_key
ent2 = MyEntity[primary_key]
```
> TypeError: Invalid count of attrs in OrgUnitClosure primary key (1 instead of 2)

**Comment by GittCatt** [comment]
Are you sure you've posted a correct example? The following works for me:
```
from pony.orm import *

db = Database()

class MyEntity(db.Entity):
    a = Required(str)
    b = Required(str)
    PrimaryKey(a, b)

db.bind(provider='sqlite', filename=':memory:')
db.generate_mapping(create_tables=True)

ent = MyEntity(a='Key1', b='Key2')
commit()

primary_key = ent.get_pk()
```

```
>>> MyEntity[primary_key]
MyEntity['Key1','Key2']
```

**Comment by paxet** [comment]
My fault, sorry.
The tuple returned from get_pk() was getting converted to str to be stored in another table, a simple 'eval' solved it.

```python
from pony.orm import *

db = Database()

class MyEntity(db.Entity):
    a = Required(str)
    b = Required(str)
    PrimaryKey(a, b)

db.bind(provider='sqlite', filename=':memory:')
db.generate_mapping(create_tables=True)

with db_session():
    ent = MyEntity(a='Key1', b='Key2')
    primary_key = str(ent.get_pk())
    ent = MyEntity[eval(primary_key)]
```

```python
>>> MyEntity[eval(primary_key)]
MyEntity['Key1', 'Key2']
```

**Comment by sashaaero** [comment]
First of all I think this issue relates to pony itself, not pony-doc repository.
About  composite keys - it was bug in Pony that we've fixed [here](https://github.com/ponyorm/pony/commit/1934624b7aa5f4106a62aa0b6bf90c94a8c395a7). And will be released soon in Pony 0.7.7
